frontend/store/wallet.ts

```diff
@@ -154,12 +154,13 @@ export const actions = {
             dispatch('authorizations/reset', undefined, { root: true });
         }
     },
-    async connect({ commit, dispatch }: ActionContext<State, State>, walletType: string): Promise<void> {
+    async connect({ commit, dispatch, rootGetters }: ActionContext<State, State>, walletType: string): Promise<void> {
         dispatch('authorizations/reset', undefined, { root: true });
         commit('setIsConnecting', true);
         try {
             const wallet = getWallet(walletType);
-            await wallet.connect();
+            const network = rootGetters['network/getMakerNetwork'];
```

Comment:
Why did you choose to use `getMakerNetwork` over `getPageNetwork` that [you've initially planned to use](https://github.com/sidestream-tech/unified-auctions-ui/pull/546#discussion_r1037359342)? 

`getMakerNetwork` is the value that "combines" both wallet and page networks, and it might be tricky to use here